tests/unit/routes/index-test.js

```diff
@@ -0,0 +1,18 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
We won't need any route tests. Routes are generally tested in acceptance tests when you run the line `visit('/')` and then assert that the current URL is what you expect. We can delete this file.